### PR TITLE
update resource-quota docs

### PR DESCRIPTION
**Pull request by HaiyangDING**
After resource QoS has been implemented, many docs have updated. However, I found these docs relating to resource quota are out-of-date. This PR tries to fix them.

This is part of #13074

### CODE REVIEW

**Comment by gmarek** [comment]
cc @bgrant0607, please reassign

**Comment by derekwaynecarr** [comment]
I will review this change since I did the related work in question.

File: docs/admin/resource-quota.md [hunk 1]
```diff
@@ -40,15 +40,20 @@ Resource quotas are a tool for administrators to address this concern. Resource
 work like this:
 - Different teams work in different namespaces.  Currently this is voluntary, but
   support for making this mandatory via ACLs is planned.
-- Users put [compute resource limits](../user-guide/compute-resources.md) on their pods.
 - The administrator creates a Resource Quota for each namespace.
+- Users put [compute resource *requests*](../user-guide/compute-resources.md) on their pods.
+  The sum of all resource *requests* of pods belonging to the same namespace should not surpass
+  the Resource Quota for the namespace. Note that we used to verify Resource Quota by checking
+  resource *limits* of the pods, but we altered to *requests* in
```
Comment (derekwaynecarr) [inline docs/admin/resource-quota.md]: I think I would rephrase this to be more like the following.

Users put compute resource requests on their pods.  The sum of all resource requests across all pods in the same namespace must not exceed any hard resource limit in any Resource Quota document for the namespace.  Note that we used to verify Resource Quota by taking the sum of resource limits of the pods, but this was altered to use resource requests.  Backwards compatibility for those pods previously created is preserved because pods that only specify a resource limit have their resource requests defaulted to match their defined limits.  The user is only charged for the resources they request in the Resource Quota versus their limits because the request is the minimum amount of resource guaranteed by the cluster during scheduling.  For more information on over commit, see http://releases.k8s.io/HEAD/docs/user-guide/compute-resources.md

File: docs/user-guide/resourcequota/README.md [hunk 2]
```diff
@@ -114,71 +118,73 @@ What happened?  I have no pods!  Let's describe the replication controller to ge
...
+  FirstSeen	LastSeen	Count	From				SubobjectPath	Reason		Message
+  42s		11s		3	{replication-controller }			FailedCreate	Error creating: Pod "nginx-" is forbidden: Must make a non-zero request for memory since it is tracked by quota.
 ```
 
 The Kubernetes API server is rejecting the replication controllers requests to create a pod because our pods
-do not specify any memory usage.
+do not specify any memory usage *request*.
 
-So let's set some default limits for the amount of cpu and memory a pod can consume:
+So let's set some default vaules for the amount of cpu and memory a pod can consume:
```
Comment (derekwaynecarr) [inline docs/user-guide/resourcequota/README.md]: So let's set some default values

**Comment by HaiyangDING** [comment]
@derekwaynecarr 

PR is updated. PTAL.

**Comment by derekwaynecarr** [comment]
LGTM
